src/open_inwoner/conf/base.py

```diff
@@ -826,6 +827,21 @@
 MOZILLA_DJANGO_OIDC_DB_CACHE = "oidc"
 MOZILLA_DJANGO_OIDC_DB_CACHE_TIMEOUT = 1
 
+#
+# 2FA SMS Verification
+#
+
+MESSAGE_BIRD_ACCESS_KEY = os.getenv("MESSAGE_BIRD_ACCESS_KEY")
+ACCOUNTS_USER_TOKEN_EXPIRE_TIME = 300
+ACCOUNTS_SMS_MESSAGE = "Open Inwoner Platform Bevestigingscode: {token} (deze code is geldig voor 5 minuten)"
+ACCOUNTS_SMS_GATEWAY = {
+    "BACKEND": config(
+        "ACCOUNTS_SMS_GATEWAY_BACKEND", "open_inwoner.accounts.gateways.Dummy"
+    ),
+    "API_KEY": config("ACCOUNTS_SMS_GATEWAY_API_KEY", "openinwoner"),
+    "ORIGINATOR": config("ACCOUNTS_SMS_GATEWAY_ORIGINATOR", "openinwoner"),
+}
+
```

Comment:
I see both MESSAGE_BIRD_ACCESS_KEY and ACCOUNTS_SMS_GATEWAY_API_KEY which feels suspicious, as I don't see the MESSAGE_BIRD_ACCESS_KEY being used.